app/views/groups/index.html.erb

```diff
@@ -1,5 +1,24 @@
 <h1>All groups</h1>
 
+<script type="text/javascript">
+  $(document).ready(function(){
+    $('#mySel').change(function(){
+      var value = $(this).val();
+      $('tr').show();
+      if (value != 'all') {
+        $('tr').not('.' + value).hide();
+      }
+    });
+  });
+</script>
+
+<select id="mySel">
+  <option selected="selected" value="all"> All </option>
+  <%= @neighborhoods.each do |nh| %>
```

Comment:
@HacksForHugs You can't be blamed for this, but we don't want inline `<script>` tags within HTML files. We'll put this jQuery in a dedicated js file elsewhere (will discuss in Tuesday meeting).